snap/snapcraft.yaml

```diff
@@ -19,25 +19,41 @@ hooks:
 
 environment:
   JAVA_HOME: $SNAP/usr/lib/jvm/java-18-openjdk-amd64
-  LOG_DIR: $SNAP_COMMON/var/log/kafka
 
 slots:
-  logs:
+  kafka-logs:
     interface: content
     source:
       read:
         - $SNAP_COMMON/var/log/kafka
+  cc-logs:
+    interface: content
+    source:
+      read:
+        - $SNAP_COMMON/var/log/cruise-control
 
 apps:
   daemon:
```

Comment:
nit: I would change this to be `kafka`. Makes things explicit once we start having multiple apps on the snap.